package.json

```diff
@@ -34,7 +34,11 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.6",
```

Comment:
Any reason why we need to add emotion here? I don't think we need it. Or is it a requirement of next-themes?

Reply:
it's a peer of `@mui` icons